app/services/comment_creation_service.rb

```diff
@@ -0,0 +1,12 @@
+class CommentCreationService
+  def initialize(params)
+    @user = params[:user]
+    @post_id = params[:post_id]
+    @text = params[:text]
+  end
+
+  def create_comment!
```

Comment:
I tend to prefer Ruby core's naming convention of not having a `bang!` method if there's no `nonbang` variant.